benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -300,12 +300,52 @@ def save_json_results(args: argparse.Namespace, benchmark_result, server_metrics
     "config": {
       "model": args.model,
       "model_server": args.backend,
-      "start_time": start_dt_proto.ToJsonString(),
+      # "start_time": start_dt_proto.ToJsonString(), TODO: Marshall this such that the unmarshaller doesnt error
```

Comment:
It would be good to have some time here even if it is not entirely accurate so we have some way to know when it was run. Maybe datetime.now.strftime()?